stack.yaml

```diff
@@ -0,0 +1,69 @@
+# This file was automatically generated by 'stack init'
```

Comment:
I want to keep cabal-install as a primary build tool for this repo. Do we really need it here?